Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2891,6 +2894,22 @@ def init_globals(command_name: str = ''):
         CACHED_INTEGRATION_CONTEXT = get_integration_context(SYNC_CONTEXT)
 
 
+def parse_common_channels(common_channels: str):
+    common_channels = (common_channels or '').strip()
```

Comment:
```suggestion
def parse_common_channels(common_channels: str = ''):
    common_channels = (common_channels).strip()
```

Reply:
this doesnt work in the case you got None as the parameter, since it is called explicitly

Reply:
So isn't it better to change
```
common_channels = demisto.params().get('common_channels', None)
````
to
```
common_channels = demisto.params().get('common_channels', '')
```?
Anyway parse_common_channels is expecting a str, and None is not a str.